config.json

```diff
@@ -1,8 +1,9 @@
 {
     "platform": "virtualbox",
-    "box_name": "playa_mesos_ubuntu_14.04",
+    "box_name": "mesos_026_ubuntu_14.04",
     "base_url": "http://downloads.mesosphere.io/playa-mesos",
     "ip_address": "10.141.141.10",
+    "mesos_release": "0.26.0-0.2.145.ubuntu1404",
```

Comment:
Updated the base Virtualbox image instead to include Mesos 0.26.0 and Marathon 0.13.0. Can you take a look and see if this meets your needs? Would prefer to manage through image updates and not pinning in the build config.